tests/unit/authors/test_authors_utils.py

```diff
@@ -46,3 +49,66 @@ def test_that_bai_conforms_to_the_spec():
     assert bai("Müller, Andreas") == "A.Mueller"
     assert bai("Hernández-Tomé, G.") == "G.Hernandez.Tome"
     assert bai("José de Goya y Lucientes, Francisco Y H") == "F.Y.H.Jose.de.Goya.y.Lucientes"
+
+
+def test_get_author_record_from_xml_response():
+    xml = """<?xml version="1.0" encoding="UTF-8"?>\n
```

Comment:
@drjova  do you think I should use fixtures here, or leave it like this to be more readable as the XMLs are quite small?